packages/react/src/components/form/form.tsx

```diff
@@ -28,20 +27,11 @@ export const Form: React.FC<Props> = ({
   text,
 }) => {
   const flowState = useFlowState({ onSuccess });
-  const { theme, showBanner } = useConfiguration();
```

Comment:
I think to truly deprecate this we need to still support this for some time before removing it

I guess something like:
```ts
const { theme } = useConfiguration()
const root = useRootThemeProps()

const t = theme ?? root.theme
const cn = `${className} ${root.className}`
``` 

and then duplicate the `<ThemeRoot>` implementation to be in this component too.